src/main/java/core/basesyntax/serviceimpl/ShopServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.serviceimpl;
```

Comment:
The package statement should not be followed by an empty line. Please remove the empty line after the package statement.